applications/dashboard/src/scripts/compatibilityStyles/inputStyles.ts

```diff
@@ -261,6 +267,78 @@ export const inputCSS = () => {
             all: globalVars.meta.spacing.default,
         }),
     });
+
+    const formSpacer = 8;
+
+    cssOut(`.StructuredForm .P`, {
+        ...margins({
+            vertical: 15,
```

Comment:
Is this intentionally 15? It feels a bit out of place. Any chance we could use our gutter/half gutter or some other existing spacing value here?